recipes-core/systemd/systemd-conf_%.bbappend

```diff
@@ -18,3 +18,5 @@ do_install_append_imx7d-pico() {
 	install -Dm 0644 ${WORKDIR}/eth.network ${D}${sysconfdir}/systemd/network/eth.network
 	install -Dm 0644 ${WORKDIR}/rndis.network ${D}${sysconfdir}/systemd/network/rndis.network
 }
+
+FILES_${PN}_append = " /etc/systemd/network"
```

Comment:
Should be `${sysconfdir}/systemd/network/` instead.

But, I think we have to use the same path as the main recipe -> https://github.com/openembedded/openembedded-core/blob/master/meta/recipes-core/systemd/systemd-conf_244.1.bb#L33

Reply:
indeed, it's better solution ;)